src/HomeWork/Sum/SumMain.java

```diff
@@ -1,33 +1,34 @@
 package HomeWork.Sum;
 
-import HomeWork.NumbRevers.ReversMethod;
-
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class SumMain {
     public static void main(String[] args) {
-        boolean miss = false;
-        System.out.print("Enter the numbers that you want to sum: ");
-        int sum =0;
-        int numbers = 0;
-
+        boolean isExit;
         do {
-            Scanner sc = new Scanner(System.in);
-            numbers = sc.nextInt();
-            sum+= numbers;
-
-            }while (numbers!=0);
             try {
-                System.out.println("Sum" + sum);
-
-
+                System.out.println("Enter the required size of the array : ");
+                Scanner s = new Scanner(System.in);
+                int size = s.nextInt();
+                int myArray[] = new int[size];
```

Comment:
It seems to me that 'myArray' isn't a good name for the array, this name doesn't explain the purpose of its creation.  
Maybe let's rename it to 'numbersFromConsole', 'enteredNumbers' or smth like this